pkg/ivd/ivd_protected_entity.go

```diff
@@ -56,6 +56,14 @@ type metadata struct {
 }
 
 func (this IVDProtectedEntity) GetDataReader(ctx context.Context) (io.ReadCloser, error) {
+	this.logger.Infof("GetDataReader called on IVD Protected Entity, %v", this.id.String())
```

Comment:
This ReloadConfig might not be necessary, since vcenter connect is explicitly called inside `getDiskConnectionParams`. Please double check.

Reply:
The point was to make connection checks on all external-facing APIs. `getDiskConnectionParams` performs a `vc.Connect(ctx)`, so the assumption is `vc` is initialized, if there were issues during initialization this would be nil.